src/Sarif/Visitors/SarifVersionOneToCurrentVisitor.cs

```diff
@@ -1,20 +1,86 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using System;
+using System.Collections.Generic;
+using System.Linq;
 using Microsoft.CodeAnalysis.Sarif.VersionOne;
 
 namespace Microsoft.CodeAnalysis.Sarif.Visitors
 {
     public class SarifVersionOneToCurrentVisitor : SarifRewritingVisitorVersionOne
     {
+        public SarifLog SarifLog { get; private set; }
+
         public override SarifLogVersionOne VisitSarifLogVersionOne(SarifLogVersionOne node)
         {
             SarifLog = new SarifLog();
+            SarifLog.Runs = new List<Run>();
+            SarifLog.Version = SarifVersion.TwoZeroZero;
+
+            foreach (RunVersionOne run in node.Runs)
+            {
+                VisitRunVersionOne(run);
+            }
 
-            return base.VisitSarifLogVersionOne(node);
+            return null;
         }
 
-        public SarifLog SarifLog { get; private set; }
+        public override RunVersionOne VisitRunVersionOne(RunVersionOne node)
+        {
+            if (node != null)
+            {
+                Run run = new Run()
+                {
+                    Architecture = node.Architecture,
+                    BaselineId = node.BaselineId,
+                    AutomationId = node.AutomationId,
+                    DefaultFileEncoding = "???",
```

Comment:
??? Just placeholders, assuming these properties will be default

Reply:
I don't understand this comment. Also, if there is something to be explained here, let's put the comment in the code.

Reply:
Just noting that I'm not going to leave it that way. I will push new changes.